client/util/test_data/certificate_config/homeExpansion.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
nit: I would name this file certificate_config_home_expansion.json and put it into test_data folder directly.